REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,131 @@
+
+Criteria for ```INCUBATING``` status include:
+* Supported by at least two committers
+* Basic documentation is provided
+* Minimal CI/CD pipelines are implemented (when applicable)
+
+### ```SANDBOX```
+
+Sandbox repositories are in the earliest stage of maturity. They are not recommended to be used in production, but could be used by those who want to experiment with new features. Very little to no support is provided.
+
+### ```DEPRECATED```
+
+Deprecated repositories are those that are no longer maintained and are only kept for historical purposes. They are not recommended for use by adoptors. They are typically [archived](#archival) and read-only.
+
+## Lifecycle
+
+### Addition
+
+Repositories of type ```Ecosystem```, ```Infra```, or ```Special``` can be added into the GitHub organization at the discretion of the Cilium committers (in other words, by lazy consensus).
+
+Any new repository of any type must be added into the appropriate list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md) and be assigned a status (if applicable).
+
+#### Adding ```Core``` repositories
+
+Given their importance to the project, repositories of the ```Core``` type must be added through a more formal process:
+  - The new project is proposed by a committer via a GitHub issue discussion in one of the existing projects in the organization. Part of the discussion should include whether the proposed project [meets the requirements](#stable) to be assigned a ```STABLE``` status.
+  - The discussion is open to all, and people in the community are encouraged to participate. While voting is exclusive to committers, this discussion is useful for those with voting power to understand the perception of the wider Cilium community.
+  - Once sufficient discussion has taken place (for example, 3-5 business days), the committers decide if it should be added via a simple majority vote.
+  - If the proposed project is accepted and added to the Cilium GitHub organization, it is then added to the list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#core-scope) and is assigned the ```STABLE``` status.
+
+### Change of scope or status
+
+As a repository increases or decreases in activity, it may become appropriate to pursue a change in status or scope. For most cases, this can be done via lazy consensus by a Cilium committer. Particularly careful consideration should be taken when changing a status from ```INCUBATING``` to ```STABLE```.
+
+The only exception to this process is when pursuing a ```Core``` scope, upon which the process described in [Adding ```Core``` repositories](#adding-core-repositories) is followed.
+
+#### Archival
```

Comment:
maybe we should do a issue template similar to https://github.com/falcosecurity/evolution/issues/new?assignees=&labels=kind%2Fdeprecated&projects=&template=repo_deprecation_request.md&title=

Reply:
Then I would also say it is lazy consensus from the project maintainers